package.json

```diff
@@ -98,16 +98,15 @@
     "@material-ui/pickers": "^4.0.0-alpha.12",
     "@mui/icons-material": "^5.8.4",
     "@mui/lab": "^5.0.0-alpha.92",
-    "@mui/x-date-pickers": "^5.0.0-beta.2",
     "@mui/material": "^5.9.2",
     "@mui/styles": "^5.9.2",
+    "@mui/x-date-pickers": "^5.0.0-beta.2",
+    "@selectquotelabs/sqhooks": ">= 2",
     "ag-grid-community": "^25.2.1",
     "ag-grid-react": "^25.2.0",
-    "@selectquotelabs/sqhooks": ">= 2",
     "moment": "^2.29.1",
     "react": ">= 17.0.0 < 18",
     "react-dom": ">= 16.8.0 < 18",
-    "scplus-shared-components": ">= 10",
```

Comment:
This may have been an accidental removal, but don't we want this shared-components declaration?

Reply:
we do, good catch!